src/Common/src/Microsoft/Win32/SafeHandles/SafeCloseIcmpHandle.cs

```diff
@@ -1,9 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.Win32.SafeHandles;
```

Comment:
This SafeHandle is specific to ICMP and so System.Net. I agree with @pgavlin: we should keep code related to networking in the same folder.

I think a better destination would be the Interop folder where other related constructs exist.